operations/configuration_management/Management_Node_Image_Customization.md

```diff
@@ -0,0 +1,163 @@
+# Management Node Image Customization
+
+**NOTE:** Some of the documentation linked from this page mentions use of the Boot Orchestration Service (BOS). The use of BOS
+is only relevant for booting compute nodes and can be ignored when working with NCN images.
+
+This document describes the configuration of a Kubernetes NCN image. The same steps could be used to modify a Ceph NCN image.
+
+1. (`ncn-mw#`) Locate the NCN image to be modified.
+
+    This example assumes that the administrator wants to modify the Kubernetes image that is currently in use by NCNs. However, the steps are the same for any NCN SquashFS image.
```

Comment:
Probably not necessary to do here, but in general there are a lot of references to "NCNs" that should probably be "management NCNs". In this case, maybe "Kubernetes NCNs" would be more accurate. Anyway, maybe it makes more sense to do this kind of audit all at once across all of docs-csm.